### PR TITLE
chore: remove cluster v1 page [DET-5163]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

We've had the new cluster page out for a few releases now, it's time to say goodbye to the old one.

## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/pages/ClusterV1.tsx [hunk 1]
```diff
@@ -1,58 +0,0 @@
-import React, { useEffect, useMemo, useState } from 'react';
-
-import Grid from 'components/Grid';
-import Message from 'components/Message';
-import Page from 'components/Page';
-import ResourceChart from 'components/ResourceChart';
```
Comment (apizzini) [inline webui/react/src/pages/ClusterV1.tsx:6]: what about deleting `ResourceChart` code?

File: webui/react/src/pages/ClusterV1.tsx [hunk 2]
```diff
@@ -1,58 +0,0 @@
...
-import Spinner from 'components/Spinner';
-import Agents from 'contexts/Agents';
-import { Resource, ResourceType } from 'types';
-import { categorize } from 'utils/data';
```
Comment (apizzini) [inline webui/react/src/pages/ClusterV1.tsx:10]: same for `categorize`, was used only here.

**Review by apizzini — APPROVED** [review]
lgtm, just made a couple of suggestions about other code that potentially can be removed since used only in the old cluster page.

File: webui/react/src/pages/ClusterV1.tsx (hunk 2 above)
Comment (hamidzr) [reply webui/react/src/pages/ClusterV1.tsx:10]: 🤔  this one I think we'll use and it shouldn't add to our bundle size as long as we don't import and use it anywhere. If you prefer we don't keep it I'll remove it

Comment (apizzini) [reply webui/react/src/pages/ClusterV1.tsx:10]: totally fine to keep!

**Pull request merged**